branchinfo.txt

```diff
@@ -3,7 +3,7 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=3
+PATCH_VERSION=4
 RELEASE_SUFFIX=rc4
```

Comment:
That can stay as is. it shows up in the version, but when we release we don't use the suffix at all. I don't see a reason to change it.